src/main.rs

```diff
@@ -325,8 +325,8 @@ fn parse_bors_reviewer(
                     .chars()
                     .all(|c| c.is_alphabetic() || c.is_digit(10) || c == '-' || c == '_')
                 {
-                    panic!(
-                        "to_author for {} contained non-alphabetic characters: {:?}",
+                    eprintln!(
```

Comment:
I think we should include = in the list above, in general this is trying to prevent us accidentally going "too far" and pulling in the commit message or so.

Reply:
Sure, sounds good!